README.md

```diff
@@ -12,8 +12,7 @@
 
 ## License
 
-This is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
-
+This SAFE Network library is dual-licensed under the Modified BSD ([LICENSE-BSD](LICENSE-BSD) https://opensource.org/licenses/BSD-3-Clause) or the MIT license ([LICENSE-MIT](LICENSE-MIT) https://opensource.org/licenses/MIT) at your option.
```

Comment:
@pravarseth , shall we consider a small change here as this is not really a library but a SAFE Network application?

Reply:
@bochaco so are you suggesting we change it to:
"This SAFE Network `application` is dual-licensed ..."

Reply:
yes @nadiaburborough , so it's more accurate